docs_2.0/manual_EN/5.deployment-and-installation/1.resource-preparations.md

```diff
@@ -51,6 +51,102 @@ You must have the correct version of the software listed below to compile Nebula
+                         gcc \
+                         gcc-c++ \
+                         cmake \
+                         gettext \
+                         curl \
+                         redhat-lsb-core
+        # For CentOS 8+, RedHat 8+, and Fedora, install libstdc++-static, libasan as well
+        $ yum install -y libstdc++-static libasan
+        ```
+
+   * For Debian and Ubuntu users, run the following commands.
+
+        ```bash
+        $ apt-get update
+        $ apt-get install -y make \
+                             m4 \
+                             git \
+                             wget \
+                             unzip \
+                             xz-utils \
+                             curl \
+                             lsb-core \
+                             build-essential \
+                             libreadline-dev \
+                             ncurses-dev \
+                             cmake \
+                             gettext
+        ```
+
+2. Check if the GCC and cmake on your host are in the right version. See [Software requirements for compiling Nebula Graph](#software-requirements-for-compiling-nebula-graph) for the required versions.
+
+    ```bash
+    $ g++ --version
+    $ cmake --version
+    ```
+
+    If your GCC and cmake are in the right version, then you are all set. If they are not, follow the sub-steps as follows.
+
+    1. Clone the nebula-common repository to your host.
+
+        ```bash
+        $ git clone https://github.com/vesoft-inc/nebula-common.git
+        ```
+
+    2. Make `nebula-common` the current working directory.
+
+        ```bash
+        $ cd nebula-common
+        ```
+
+    3. Run the following commands to install and enable CMake and GCC.
+
+        ```bash
+        # Install CMake.
+        $ ./third-party/install-cmake.sh cmake-install
+        CMake has been installed to prefix=cmake-install
+        Run 'source cmake-install/bin/enable-cmake.sh' to make it ready to use.
+        Run 'source cmake-install/bin/disable-cmake.sh' to disable it.
+
+        # Enable CMake
+        $ source cmake-install/bin/enable-cmake.sh
+
+        # Install GCC. Installing GCC to /opt requires root privilege, you can change it to other locations.
+        $ ./third-party/install-gcc.sh --prefix=/opt
+        GCC-7.5.0 has been installed to /opt/vesoft/toolset/gcc/7.5.0
+        Performing usability tests
+        Performing regular C++14 tests...OK
+        Performing LeakSanitizer tests...OK
+        Run 'source /opt/vesoft/toolset/gcc/7.5.0/enable' to start using.
+        Run 'source /opt/vesoft/toolset/gcc/7.5.0/disable' to stop using.
+
+        #Enable GCC. Please note that the path and specific version might be different from your environment.
+        $ source /opt/vesoft/toolset/gcc/7.5.0/enable
+        ```
+
 ## Requirements and suggestions for installing Nebula Graph in test environments
 
 ### Hardware requirements for test environments
```

Comment:
line 134. some problems:

```
the capacity requirement for running a Nebula Graph cluster
```

Storage for a cluster - > Disk space for a cluster.


```
<the sum of edge number and vertex number> * <average bytes of attributes> * 6 * 120%
[<the sum of edge number and vertex number> * 15 bytes + <the number of RocksDB instances> * (write_buffer_size * max_write_buffer_number) + rocksdb_block_cache] * 120%
```

please use <> to brace long keywords